### PR TITLE
1.9 Changelog and Migration Guide

### PR DESCRIPTION
**Pull request by OwenKephart**
## Changelog

NOCHANGELOG

### CODE REVIEW

**Comment by OwenKephart** [comment]
* **#25656** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/25656?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @OwenKephart and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,94 @@
...
+- [experimental] You can now define custom subclasses of `AutomationCondition` to execute arbitrary Python code in the context of a broader expression. This allows you to compose built-in conditions with custom business logic.
+- The `target` arguments on schedules and sensors are now marked stable, allowing a stable way for schedules and sensors to target asset selections without needing to define a job.
+
+### Integrations
+
+- Introduced a slate of integrations with business intelligence (BI) tools, enabling dashboards, views, and reports to be represented in the Dagster asset graph.
+  - Power BI [docs]
+  - Looker [docs]
+  - Tableau [docs]
+  - Sigma [docs]
```
Comment (OwenKephart) [inline CHANGES.md:21]: @benpankow @maximearmstrong ping to update this (a comment on this PR works)

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,94 @@
...
+- Pipes 1.9 changes: TODO
```
Comment (OwenKephart) [inline CHANGES.md:22]: @danielgafni @smackesey ping to update this

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,94 @@
...
+- Rich asset metadata in ELT integrations: TODO
```
Comment (OwenKephart) [inline CHANGES.md:23]: @benpankow @maximearmstrong ping to update this

File: CHANGES.md [hunk 4]
```diff
@@ -1,5 +1,94 @@
...
+
+### Airlift
+
+- TODO
```
Comment (OwenKephart) [inline CHANGES.md:27]: @dpeng817 @schrockn ping to update this

File: MIGRATION.md [hunk 5]
```diff
@@ -2,6 +2,37 @@
...
+
+## Notable behavior changes
+
+- Backfills have been moved from their own tab underneath the Overview page to entries within the table on the Runs page. This reflects the fact that backfills and runs are similar entities that share most properties. You can continue to use the legacy Runs page with the “Revert to legacy Runs page” user setting. ([GitHub Discussion](https://github.com/dagster-io/dagster/discussions/24898))
+- By default, `AutomationConditionSensorDefinitions` will now emit backfills to handle cases where more than one partition of an asset is requested on a given tick. This allows that asset's `BackfillPolicy` to be respected. This feature can be disabled by setting `allow_backfills` to `False` on the sensor definition.
+
+## Deprecations
+
+- LookerResource? [superseded]
```
Comment (OwenKephart) [inline MIGRATION.md:18]: @maximearmstrong ping to update this

File: CHANGES.md [hunk 6]
```diff
@@ -1,5 +1,94 @@
...
+- You can now assign `AutomationConditions` to asset checks, via the `automation_condition` parameter on `@asset_check` or `AssetCheckSpec`.
+- You can now assign `AutomationConditions` to observable source assets, via the `automation_condition` parameter on `@observable_source_asset`.
+- [experimental] You can now define custom subclasses of `AutomationCondition` to execute arbitrary Python code in the context of a broader expression. This allows you to compose built-in conditions with custom business logic.
+- The `target` arguments on schedules and sensors are now marked stable, allowing a stable way for schedules and sensors to target asset selections without needing to define a job.
```
Comment (OwenKephart) [inline CHANGES.md:13]: @smackesey ping to confirm wording

File: CHANGES.md (hunk 2 above)
Comment (OwenKephart) [reply CHANGES.md:22]: or maybe @yuhan

File: CHANGES.md (hunk 6 above)
Comment (smackesey) [reply CHANGES.md:13]: lgtm

File: CHANGES.md (hunk 2 above)
Comment (danielgafni) [reply CHANGES.md:22]: @OwenKephart do we want more here? like a short overview of Pipes? Or that's fine?

File: CHANGES.md (hunk 1 above)
Comment (sryza) [reply CHANGES.md:21]:
```
- [Looker](https://docs.dagster.io/integrations/looker)
- [Power BI](https://docs.dagster.io/integrations/powerbi)
- [Sigma](https://docs.dagster.io/integrations/sigma)
- [Tableau](https://docs.dagster.io/integrations/tableau)
```

File: CHANGES.md (hunk 3 above)
Comment (maximearmstrong) [reply CHANGES.md:23]: Rich asset metadata in ELT integrations:
- the `dagster/relation_identifier` metadata tag has been renamed to `dagster/table_name`
- the `dagster/table_name` metadata tag has be added for Airbyte, dlt and Sling assets. The value for this tag is the fully qualified named of the destination model.

File: CHANGES.md (hunk 2 above)
Comment (OwenKephart) [reply CHANGES.md:22]: @danielgafni A quick explanation of why we think Pipes is a superior interface (like 1-2 sentences) would be great

File: CHANGES.md [hunk 7]
```diff
@@ -1,5 +1,94 @@
...
+
+## Changes since 1.8.13 (core) / 0.24.13 (libraries)
+
+### New
+
+- The `tags` parameter to `@asset` and `AssetSpec` is no longer marked as experimental.
+- The `@observable_source_asset` decorator now supports an `automation_condition` argument.
+- `AutomationCondition` and associated APIs are no longer marked as experimental.
+- Added a new `use_user_code_server` parameter to `AutomationConditionSensorDefinition`. If set, the sensor will be evaluated in the user code server (as traditional sensors are), allowing custom `AutomationCondition` subclasses to be evaluated.
+- Added a new column to the BulkActions table and a new BackfillTags table to improve performance when filtering Backfills. To take advantage of these performance improvements, run `dagster instance migrate`. This migration involves a schema migration to add the new column and table, and a data migration to populate the new column for historical backfills.
```
Comment (jamiedemaria) [inline CHANGES.md:45]: Can we combine this entry and the one below "Added a new column to the Runs table ..." to 

- Added a new column to the BulkActions table, a new column to the Runs table, and a new BackfillTags table to improve the performance of the Runs page. To take advantage of these performance improvements, run `dagster instance migrate`. This migration involves a schema migration to add the new columns and table, and a data migration to populate the new columns for historical backfills and runs.

**Review by yuhan — APPROVED** [review]

**Pull request merged**